### PR TITLE
Add Hologram Support to AOneBlock Phases

### PR DESCRIPTION
**Pull request by Fredthedoggy**
This Commit Adds Hologram Support (via HolographicDisplays) to Phases in the same way FixedBlocks work.
```yaml
  holograms:
    0: "&aGood Luck!"
    1: "&aAnd Have Fun!"
```

This requires BentoBoxWorld/BentoBox#1794 for HolographicDisplays.

The other thing I will note is that due to the *first* block placed when the gamemode starts not being in any phase (Break 1 block to unlock plains) I've had to add the Starting hologram to the Locale file. It doesn't make very much sense, but thats the only good way I've found. I'm open to other suggestions though :)

```yaml
aoneblock:
  island:
    starting-hologram: "&aWelcome to AOneBlock\n&eBreak This Block to Begin"
```

### CODE REVIEW

**Comment by Fredthedoggy** [comment]
ah I messed that merge up, sorry!

**Comment by tastybento** [comment]
So, just to clarify - does it show Good Luck! on the first block broken, and then "And Have Fun!" when you break the second block?

**Comment by Fredthedoggy** [comment]
> So, just to clarify - does it show Good Luck! on the first block broken, and then "And Have Fun!" when you break the second block?

When you first create the island (eg. after `/is create`) it shows `"&aWelcome to AOneBlock\n&eBreak This Block to Begin"`, then when you break one block it shows `Good Luck!` (at the same time as the "0 Block" from the phase is placed, then when you break a second block it shows "Have Fun" (same as "1 Block")

**Comment by BONNe** [comment]
Isn't its better to add it as an addon, or will it be too complicated to take apart from AOneBlock core?

**Comment by Fredthedoggy** [comment]
Its *entirely* based around the concept of OneBlock, so I think it should be here. Since it changes when you break the block, and acts as a "tutorial" if you want

File: src/main/java/world/bentobox/aoneblock/AOneBlock.java [hunk 1]
```diff
@@ -92,22 +98,44 @@ public void onEnable(){
...
+                        if (hololine != null) {
+                            final Hologram hologram = HologramsAPI.createHologram(BentoBox.getInstance(), island.getCenter().add(0.5, 2.6, 0.5));
+                            for (String line : hololine.split("\\n")) {
+                                hologram.appendTextLine(ChatColor.translateAlternateColorCodes('&', line));
+                            }
+                        }
+                    }
+                }
+            }
+        }, 400L); // Wait 20 Seconds so that all Islands are Loaded
```
Comment (tastybento) [inline src/main/java/world/bentobox/aoneblock/AOneBlock.java:138]: This is a bit of a hack. You can override the Addon#allLoaded method instead. It gets called when BentoBox is fully loaded and all addons loaded.

**Review by tastybento — CHANGES_REQUESTED** [review]
See my comment - best to use the method instead of guessing a time to load.

**Pull request merged**